solidity/random-beacon/test/fixtures/index.ts

```diff
@@ -47,6 +47,8 @@ export const params = {
   unauthorizedSigningSlashingAmount: to1e18(100000),
   minimumAuthorization: to1e18(100000),
   authorizationDecreaseDelay: 0,
+  reimbursmentPoolStaticGas: 37500,
```

Comment:
Static gas cost includes the execution of code in the `refund()` + transaction cost. This number was captured at the moment of implementing the tests a while ago. After addressing some of the comments the avg. gas bumped to ~41k for the `refund()`. The static gas will be passed in the constructor during the deployment and should be probably revisited then.

```
gas for refund (min, max, avg) 40433 | 41893 | 41075  
```

I don't think we can precisely tell what is the cost of the `refund()`. We have min, max and avg gas usage depending on the situation. If we assume that the avg. is 41k then at most a caller will be net-negative around 800-900gas. Assuming the gas is 100gwei that is around 0,00008 ETH. Or can be a bit net-positive if a user hits a min gas path.

More integration tests between ECDSA Wallet registry / random-beacon and Reimbursement pool will be added in the next PRs where the refunds are checked.

Reply:
It would be good to establish this number before merging this PR. `ReimbursmentPool` code should not change in the future so there is no better moment than now, especially that as you say, we will have more tests added in the following PRs and we should be able to validate our choice this way.

I think the strategy should be: if we can't use a precise number, we establish one that is returning a tiny bit more than actually spent. If we see a frontrunning in the network, we will lower it and everyone will receive a tiny bit less than what they have actually spent.